samples/java_layout/src/main/java/io/customer/android/sample/java_layout/data/PreferencesStoreUtils.java

```diff
@@ -0,0 +1,51 @@
+package io.customer.android.sample.java_layout.data;
+
+
+import androidx.annotation.MainThread;
+import androidx.datastore.preferences.core.MutablePreferences;
+import androidx.datastore.preferences.core.Preferences;
+import androidx.datastore.preferences.core.PreferencesKeys;
+import androidx.datastore.rxjava3.RxDataStore;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.core.Single;
+
+class PreferencesStoreUtils {
```

Comment:
Maybe the methods in here can be part of `PreferencesDataStoreImpl` instead of separate utils class? that is reliant on `RxDataStore<Preferences>`  as well ?

Reply:
I did this to shorten class length and avoid code repetition. Moving this back to `PreferencesDataStore` will add little too much code to the class.